### DISCUSSION TITLE
実行してもiPhone真っ白になってしまって次に進めません！

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#131] [Question-original] [answered]
## (1) 実現したいこと
Realmを使った家計簿アプリの作成です
## (2) 実現するために自分が試した内容とその結果
教科書の23ページまでやりました。
## (3) 調べた内容や資料
データベース(Realm)2の教科書
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
![image](https://user-images.githubusercontent.com/75009903/217498436-99f68781-79ac-46a3-8021-f95d82df3392.png)
## (5) GitHubのリンク
https://github.com/KOTARO1209/RealmKakeibo

**Comment by txxxxc** [comment, marked as answer]
tableViewの関連付けってできてる？

**Reply by KOTARO1209** [reply]
![image](https://user-images.githubusercontent.com/75009903/217836979-6d0eeddc-1bc1-4238-a0c6-69448f75ae80.png)

このOutletsのところが関連付けられていないからですかね？でも教科書にはここの関連付け書いていなさそう？です🙇‍♂️
